blocks/procedures.ts

```diff
@@ -786,9 +845,10 @@ const PROCEDURE_CALL_COMMON = {
     // Reconnect any child blocks.
     if (this.quarkIds_) {
       for (let i = 0; i < this.arguments_.length; i++) {
-        const quarkId = this.quarkIds_[i];
+        const quarkId: string = this.quarkIds_[i];  // TODO(#6920)
         if (quarkId in this.quarkConnections_) {
-          const connection = this.quarkConnections_[quarkId];
+          // TODO(#6920): investigate claimed circular initialisers.
```

Comment:
What does "circular intialisers" refer to?

Reply:
From #6920:
> `setProcedureParameters_` unexpectedly required declarations of the types of the `quarkId` and `connection` local variables avoid "implicitly has type 'any' because it does not have a type annotation and is referenced directly or indirectly in its own initializer" errors.  Since the definitions do not appear to be circular (though `connection` is initialised based on `quarkId`) this smells and probably warrants further investigation.